tests/JsonSchema/Command/JsonSchemaGenerateCommandTest.php

```diff
@@ -332,4 +334,17 @@ public function testResourceWithEnumPropertiesSchema(): void
             $properties['genders']
         );
     }
+
+    /**
+     * @group legacy
+     * TODO: find a way to keep required properties if needed
+     */
+    public function testPatchSchemaRequiredProperties(): void
+    {
+        $this->tester->run(['command' => 'api:json-schema:generate', 'resource' => 'ApiPlatform\Tests\Fixtures\TestBundle\ApiResource\PatchRequired\PatchMe', '--format' => 'json']);
+        $result = $this->tester->getDisplay();
+        $json = json_decode($result, associative: true);
+
+        $this->assertEquals(['b'], $json['definitions']['PatchMe']['required']);
+    }
```

Comment:
Please also add a test when a user wants to force `required` on a property on a Patch operation with `"skip_patch_json_schema_required_properties" => true`.